package/python-kiwi-spec-template

```diff
@@ -617,17 +618,13 @@ Provides manual pages to describe the kiwi commands
 %prep
 %setup -q -n kiwi-%{version}
 
-# Drop shebang for kiwi/xml_parse.py, as we don't intend to use it
-# as an independent script
-sed -e "s|#!/usr/bin/env python||" -i kiwi/xml_parse.py
```

Comment:
This should remain.

Reply:
yes was removed during my testing, should have never been dropped